cosmos/operators/local.py

```diff
@@ -252,6 +256,67 @@ def store_compiled_sql(self, tmp_project_dir: str, context: Context, session: Se
         else:
             self.log.info("Warning: ti is of type TaskInstancePydantic. Cannot update template_fields.")
 
+    @staticmethod
+    def _configure_remote_target_path() -> tuple[Path, str] | tuple[None, None]:
+        """Configure the remote target path if it is provided."""
+        if not remote_target_path:
+            return None, None
+
+        _configured_target_path = None
+
+        target_path_str = str(remote_target_path)
+
+        remote_conn_id = remote_target_path_conn_id
+        if not remote_conn_id:
+            target_path_schema = target_path_str.split("://")[0]
```

Comment:
It may be worth importing `from urllib.parse import urlparse` and using


```suggestion
            target_path_schema = urlparse(target_path).scheme
```

Reply:
thanks, using urlparse now.